src/components/Stickers.js

```diff
@@ -6,11 +6,12 @@ import { AcaciaSVG } from './assets/Acacia.js';
 import { ElephantGrassSVG } from './assets/ElephantGrass.js';
 
 class sticker{
```

Comment:
I don't know what convention this project follows (@reginawang3495?), but I usually capitalize class names.

Reply:
yeeee same, this repository is a bit legacy, which is why it's like this, but I agree